schema/src/db/tables.rs

```diff
@@ -316,15 +316,15 @@ mod tests {
     const CREATE_THING1: &str = concat!(
         "CREATE TABLE IF NOT EXISTS\n",
         " test_namespace.thing1 (\n",
-        " id bigserial primary key not null,\n",
+        " id bigint primary key not null,\n",
```

Comment:
- Revert this per a conversation with TJ
- We should create an issue to make this concept explicit -- and add this to the markdown book